radixdlt/src/test/java/com/radixdlt/counters/SystemCountersImplTest.java

```diff
@@ -28,15 +29,21 @@
 import static org.junit.Assert.assertNotNull;
 
 public class SystemCountersImplTest {
+
+	@After
+	public void tearDown() {
```

Comment:
Counters should be reset in a setup (@ Before) method.  As things stand at the moment, the initial conditions for the first run test are not well-defined, as it could be run after some other test that has modified the singleton.

I don't think it's necessary to reset the counters after the tests are complete, as other tests should ensure that their own initial condition requirements are met.